backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/document/location/adapter/DocumentLocationAdapterFactory.java

```diff
@@ -33,4 +33,11 @@ public interface DocumentLocationAdapterFactory
 	Optional<IDocumentDeliveryLocationAdapter> getDocumentDeliveryLocationAdapter(Object record);
 
 	Optional<IDocumentHandOverLocationAdapter> getDocumentHandOverLocationAdapter(Object record);
+
+	default Optional<IDocumentShipFromLocationAdapter> getDocumentShipFromLocationAdapter(Object record)
```

Comment:
why `default`?
how comes that the other methods are not default but only this one?